lessons/basics/enum.md

```diff
@@ -83,22 +92,36 @@ iex> Enum.map([0, 1, 2, 3], fn(x) -> x - 1 end)
 
 ### min
 
-Find the `min` value in our collection:
+`min/1` finds the `min` value in the collection.
```

Comment:
No point, just me being silly & missing a small detail. 👍 will change in both locations & push back up sometime today after I finish up the days work.